.gitlab-ci.yml

```diff
@@ -1113,3 +1103,19 @@ workflow:
       paths:
         - .go-version
       compare_to: main # TODO: use a variable, when this is supported https://gitlab.com/gitlab-org/gitlab/-/issues/369916
+
+.on_fakeintake_changes:
+  - changes:
+      - "test/fakeintake/**/*"
+      - .gitlab/fakeintake.yml
+
+.on_fakeintake_changes_main:
+  - !reference [.on_fakeintake_changes]
+  - if: $CI_COMMIT_BRANCH == "main"
+
+.on_fakeintake_changes_with_manual:
+  - changes:
+      - "test/fakeintake/**/*"
+      - .gitlab/fakeintake.yml
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
Should this be :
```suggestion
  - !reference [.on_fakeintake_changes]
```
?

Reply:
It should be, but then when I reference `on_fakeintake_changes_with_manual` it seems it doesn't work properly. That's why I duplicated this